index.html

```diff
@@ -85,16 +87,17 @@ <h1>Hey there. I’m Dhuruv <br><span class="h-green">I’m a software developer
             </div>
         </div>
     </section>
+
     <main id="main-sec">
         <div class="text">
             <h2>My Recent Works</h2>
             <img src="./icons/Indicator.png" alt="Indicator">
         </div>
         <section class="work-container">
```

Comment:
- [ ] Kindly remove all the hardcoded projects, this section is supposed to be recreated in the same way you did to your popup section, you can use the same array to dynamically generate this section, this is a project requirement
![image](https://user-images.githubusercontent.com/61408860/180065064-07733e92-1d6f-4c5a-b20b-c06906d98286.png)

Reply:
Fixed